op-chain-ops/cmd/celo-migrate/main.go

```diff
@@ -207,7 +217,7 @@ func main() {
 	log.Info("Finished migration successfully!")
 }
 
-func ApplyMigrationChangesToDB(genesis *core.Genesis, dbPath string, commit bool) (*types.Header, error) {
+func ApplyMigrationChangesToDB(ctx *cli.Context, genesis *core.Genesis, dbPath string, commit bool) (*types.Header, error) {
```

Comment:
Ideally we'd avoid passing in the context here, could we set a default?

Reply:
What's the issue with passing the context here?

Reply:
It's not wrong but has a weird taste. If we do this we wouldn't need to pass the dbPath in as a parameter either, but there we do all handling of options before calling the migration code.

Reply:
Ok got you